cmd/skr/main.go

```diff
@@ -41,7 +41,7 @@ func main() {
 	httpPort := flag.String("port", "8080", "Port on which to listen")
 	allowTestingMismatchedTCB := flag.Bool("allowTestingMismatchedTCB", false, "For TESTING purposes only. Corrupts the TCB value")
 	// NOTE: these 4 input arguments are typically only used in AKS, not ACI
-	grpcPort := flag.String("keyprovider_sock", "127.0.0.1:50000", "Port on which the grpc key provider to listen")
+	grpcPort := flag.String("keyprovider_sock", os.Getenv("KEYPROVIDER_SOCK"), "Port on which the grpc key provider to listen")

```

Comment:
to match the other flags, I would make this camel-case

Reply:
I'll ask and see, env vars are usually capital snake case. I feel like the other ones should be changed too